setup_pubsub.sh

```diff
@@ -44,7 +50,7 @@ create_subscription() {
 create_topic_and_subscription() {
   # Expects three arguments: project name, topic name, and subscription name
   create_topic "$1" "$2"
-  create_subscription "$1" "$2" "$3"
+  create_subscription "$1" "$2" "$3" "$4"
```

Comment:
Function Doc comment needs updating

Reply:
I changed it but accidentally reverted the change 🤦